### PR TITLE
added more functions

### PR DESCRIPTION
**Pull request by RuslanYaremenkoWizardsDev**

### CODE REVIEW

**Comment by SkrypnikMR** [comment]
Perfect!

File: index.js [hunk 1]
```diff
@@ -1,3 +1,22 @@
+function getSum (a, b) {
+    return a + b;
+}
+function getSum (a, b) {
+    return a + b;
+}
+function getSum (a, b) {
+    return a + b;
+}
+function getSum (a, b) {
```
Comment (RuslanYaremenkoWizardsDev) [inline index.js:10]: rename this files

File: index.js [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+    return a + b;
+}
+
+function getSum (a, b) {
+    return a + b;
+}
+function getSum (a, b) {
```
Comment (RuslanYaremenkoWizardsDev) [inline index.js:17]: too much duplicates

Comment (RuslanYaremenkoWizardsDev) [reply index.js:17]: also you narushel dry principle

**Pull request merged**